eng/pipelines/libraries/base-job.yml

```diff
@@ -107,7 +111,7 @@ jobs:
         - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
           - _buildScript: ./$(_buildScriptFileName)$(scriptExt)
           - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
-            - _stripSymbolsArg: -stripSymbols
+            - _stripSymbolsArg: ' -stripSymbols'
```

Comment:
This is wrong, you shouldn't need an space here as we add an space when passing the arguments down to the build script on the build step. Also this is only passed down when `isOfficialBuild == true` which in CI is never true, so this is not affecting your CI build. Also, remove the quotes around 'true' in the isOfficialBuild conditional as that might cause trouble in yml when evaluating booleans. I think the problem is somewhere in this condition here: 

https://github.com/dotnet/runtime/blob/f69e638fbd1af148fd4b8addd571a56a48a9be68/src/libraries/Native/build-native.proj#L23

It seems like `TargetOS` is being evaluated as != iOS for some reason. I would look at the binlog for the failed build.